src/xtgeo/common/_xyz_enum.py

```diff
@@ -5,7 +5,7 @@
 class ExtendedEnum(Enum):
     @classmethod
     def list(cls):
-        return list(map(lambda c: c.value, cls))
```

Comment:
I think i agree here, the map list-map-lambda can be a bit hard to unpack if your not used to it. The list comprehension to me, is straight forward and clear.